tests/data_snack/wrap/test_data_frame.py

```diff
@@ -0,0 +1,77 @@
+from typing import List
```

Comment:
Should we test the methods inherited form `EntityWrap`? I know it is tested there, but we assume that this dataframe version allows to do exact the same things, but we use current implementation as a shortcut, if it changes then our tests wouldn't catch it.